### DISCUSSION TITLE
Cant pull full image

### DISCUSSION THREAD

**Discussion by VasilisThePikachu** [nektos/act#2013] [Q&A]
I am using windows 10 right now and tried running it both on the command prompt terminal and in WSL (with that wsl integration in docker)

After like 10 mins of it saying it was pulling it failed to create the container cause it does not exist. Trying to pull it manually results in the error message

``failed to register layer: Error processing tar file(duplicates of file paths not supported):'``

Am I doing something wrong?

**Comment by ChristopherHX** [comment]
Can you please tell me, which layer (sha) causes this issue / number of the layer?

Also please tell me if it's ubuntu 2022 (latest) or 2020

My internet is slow and verifing the dumps takes hours of downloads on my end.


> Replacing images in progress...........
> 
> Anyone who want to go back in history to the old ubuntu 20.04/latest image here the fully qualified tags
> 
> * `catthehacker/ubuntu@sha256:598b616a8c7ce86d98ee63871cec532f4ff645125b563a8798f2ae1c98928ec7`
> * `ghcr.io/catthehacker/ubuntu@sha256:598b616a8c7ce86d98ee63871cec532f4ff645125b563a8798f2ae1c98928ec7`
> 
> They may become invalid in some months, due to garbage collection. The ghcr version is expected to last longer.

**Reply by VasilisThePikachu** [reply]
seems to happen on layer ``6b73d324fbcd`` and this would be ``full-20.04``
![image](https://github.com/nektos/act/assets/34938708/dfdfb376-898e-451f-9330-226ff624c53b)

**Reply by ChristopherHX** [reply]
Thank you for providing more information to me fixed in https://github.com/ChristopherHX/runner-image-blobs/commit/fc455b627125bbdc68614d7b8f3a0b41630673b8

Seems like I did a mistake while fixing folder permissions of the new large image

I expect that it is fixed within 4 hours now (it takes 3h to dump)

The newer large images are uploaded without docker tools, due to space limitations of hosted runners.

It tars the filesystem of the hosted runners and stream uploads them to ghcr, then in tbe old repo it is mirrored using skopeo. (the new approach allows catching up runner-image deployments)

**Reply by ChristopherHX** [reply]
I hope this problem is fixed now, I will check that again tomorrow.

**Reply by ChristopherHX** [reply]
I can pull it without issues now, ploblem fixed
